spec/paperclip/attachment_spec.rb

```diff
@@ -31,9 +31,109 @@
     file.close
   end
 
+  context "when generators and procesors are provided" do
+    class TestProcessor < Paperclip::Processor
+      def make
+        original_content = file.tap(&:rewind).read
+        file.rewind
+        result = Tempfile.new('paperclip-test')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.